client/components/mma/switch/SwitchOptions.tsx

```diff
@@ -0,0 +1,109 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	headline,
+	palette,
+	space,
+	textSans,
+} from '@guardian/source-foundations';
+import { Stack } from '@guardian/source-react-components';
+import { useContext } from 'react';
+import type { PaidSubscriptionPlan } from '../../../../shared/productResponse';
+import { getMainPlan } from '../../../../shared/productResponse';
+import { calculateMonthlyOrAnnualFromInterval } from '../../../../shared/productTypes';
+import { Card } from '../shared/Card';
+import { Heading } from '../shared/Heading';
+import type { SwitchContextInterface } from './SwitchContainer';
+import { SwitchContext } from './SwitchContainer';
+
+//TODO: this is copied from AccountOverviewV2, share it
+const pageTopCss = css`
+	margin-top: ${space[6]}px;
+	${from.tablet} {
+		margin-top: ${space[9]}px;
+	}
+`;
+
+const cardHeaderDivCss = css`
+	display: flex;
+	justify-content: space-between;
+	align-items: flex-end;
+`;
+
+const productTitleCss = css`
+	${headline.xsmall({ fontWeight: 'bold' })};
+	color: ${palette.neutral[100]};
+	margin: 0;
+	max-width: 20ch;
+
+	${from.tablet} {
+		${headline.small({ fontWeight: 'bold' })};
+	}
+`;
+
+const productSubtitleCss = css`
+	${textSans.medium({ fontWeight: 'bold' })};
+	color: ${palette.neutral[100]};
+	margin: 0;
+	max-width: 20ch;
+`;
+
+const cardHeaderCss = css`
+	${from.tablet} {
+		min-height: 0px;
+	}
+`;
+
+const pageHeadingCss = css`
+	${from.mobile} {
+		${textSans.xxlarge({ fontWeight: 'bold' })}
+	}
+`;
+
+const SwitchOptions = () => {
+	const switchContext = useContext(SwitchContext) as SwitchContextInterface;
+
+	const productDetail = switchContext.productDetail;
+	const mainPlan = getMainPlan(
+		productDetail.subscription,
+	) as PaidSubscriptionPlan;
+	const monthlyOrAnnual = calculateMonthlyOrAnnualFromInterval(
+		mainPlan.interval,
+	);
+
+	return (
+		<Stack space={3} cssOverrides={pageTopCss}>
+			<Heading cssOverrides={pageHeadingCss}>
```

Comment:
Rather than adding a CSS override it might be better to have a new prop that allows switching the header style from serif to sans-serif as we're going to want to do this on every page as well as the Account Overview. (It will probably become the default heading style everywhere eventually, although only a handful of places use this component currently.)

Reply:
I've added this as a new prop because I actually found using the cssOverride to be not so obvious - and had to set the style for the minimum breakpoint in order for it to actually override the default style.